tests/conftest.py

```diff
@@ -118,11 +119,13 @@
 from inmanta.types import JsonType
 from libpip2pi.commands import dir2pi
 
-# Import the utils module differently when conftest is put into the inmanta_tests packages
+# Import test modules differently when conftest is put into the inmanta_tests packages
 if __file__ and os.path.dirname(__file__).split("/")[-1] == "inmanta_tests":
     from inmanta_tests import utils  # noqa: F401
+    from inmanta_tests.db.common import PGRestore  # noqa: F401
```

Comment:
are these files packaged into the extension at all?

Reply:
This was indeed a bit naive of me, I hadn't checked. Luckily [they are](https://github.com/inmanta/inmanta-core/blob/279b74325e5fa9a7abd9951afc4a1f6c0b18a38f/tests_common/copy_files_from_core.py#L76)